lib/src/library_analyzer.dart

```diff
@@ -115,8 +115,14 @@ class LibraryScanner {
 
   Future<List<String>> _scanUri(String libUri) async {
     Uri uri = Uri.parse(libUri);
-    String package = uri.path.split('/').first;
-    String fullPath = _packageResolver.resolveAbsolute(uri).fullName;
+    String package = uri.pathSegments.first;
+
+    var source = _packageResolver.resolveAbsolute(uri);
```

Comment:
Was throwing w/ an NPE on `firebase_database` – at least we get a more helpful log now